compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/AwtDragAndDropManager.desktop.kt

```diff
@@ -174,140 +182,193 @@ internal class AwtDragAndDropManager(
         )
     }
 
-    private inner class ComposeTransferHandler : TransferHandler() {
+    /**
+     * Receives and processes events from the [ComposeDropTarget] installed in the root component.
+     */
+    val dropTargetListener = object : DropTargetListener {
```

Comment:
Might be private in case if `ComposeDropTarget` will receive it via lambda

Reply:
Yes, but then I'd need to create an unnecessary lambda, and hiding it here isn't important as it's all part of one file/implementation.